data/ecosystems/t/ton.toml

```diff
@@ -1029,170 +1275,385 @@ url = "https://github.com/dTelecom/ton"
 [[repo]]
 url = "https://github.com/elyahk/TonWalletContest"
 
+[[repo]]
+url = "https://github.com/EmelyanenkoK/description-config-for-TON-Blockchain"
+
 [[repo]]
 url = "https://github.com/EmelyanenkoK/jetton_dao"
 
 [[repo]]
-url = "https://github.com/ent1tydev/ton-browser-lite"
+url = "https://github.com/EmelyanenkoK/modern_jetton"
 
 [[repo]]
-url = "https://github.com/eq-lab/TonDIDHackathon"
+url = "https://github.com/EmelyanenkoK/multiowner"
 
 [[repo]]
-url = "https://github.com/evaafi/evaa-landing"
+url = "https://github.com/EmelyanenkoK/mytonctrl"
 
 [[repo]]
-url = "https://github.com/evaafi/front-end"
+url = "https://github.com/EmelyanenkoK/newton-blockchain.github.io"
 
 [[repo]]
-url = "https://github.com/evaafi/liquidation-bot-template-public"
+url = "https://github.com/EmelyanenkoK/nft-sender"
 
 [[repo]]
-url = "https://github.com/evaafi/redstone-price-verify"
+url = "https://github.com/EmelyanenkoK/nft-standard-draft"
 
 [[repo]]
-url = "https://github.com/evaafi/telegram-info-bot"
+url = "https://github.com/EmelyanenkoK/nomination-contract"
 
 [[repo]]
-url = "https://github.com/evaafi/terms_of_use"
+url = "https://github.com/EmelyanenkoK/nominator_pool"
 
 [[repo]]
-url = "https://github.com/evaafi/tonconnect-config"
+url = "https://github.com/EmelyanenkoK/OracleHub"
 
 [[repo]]
-url = "https://github.com/FairJournal/backend"
+url = "https://github.com/EmelyanenkoK/orion-exchange"
```

Comment:
can probably remove given it's more of an oracle / exchange?

Reply:
Done